arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1041 @@
+                    },
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "Access key to regenerate",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollectionAccessKey"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Get access keys completed successfully",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollectionAccessKeys"
+                        }
+                    },
+                    "default": {
+                        "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            }
+        },
+        "/providers/Microsoft.PowerBI/operations": {
+            "get": {
+                "tags": [
+                    "Operations"
+                ],
+                "operationId": "getAvailableOperations",
+                "description": "The **Get Available Operations** operation indicates which operations can be performed by the Power BI Resource Provider.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request completed successfully",
+                        "schema": {
+                            "$ref": "#/definitions/OperationList"
+                        }
+                    },
+                    "default": {
+                        "description": "Default Response. It will be deserialized as per the Error defintion specified in the schema. Exception will be thrown.",
+                        "schema": {
+                            "$ref": "#/definitions/Error"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.PowerBI/locations/{location}/operationresults/{operationId}": {
+            "get": {
+                "tags": [
+                    "Operations"
+                ],
+                "operationId": "getLongRunningOperationStatus",
```

Comment:
You don't need to expose this method on the generated client. If you add the long running operation extension on the operation then the generated code will automatically poll for the customer. We have the long running operation logic included in the runtime of every language generated by autorest